cosmos/azure-spring-data-cosmos/cosmos-multi-tenant-samples/tenant-by-container/src/main/java/com/azure/spring/data/cosmos/example/OrderController.java

```diff
@@ -0,0 +1,73 @@
+package com.azure.spring.data.cosmos.example;
+
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.spring.data.cosmos.example.tenant.TenantStorage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Controller;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+@PropertySource("classpath:application.yaml")
+@Controller
+@RequestMapping(path = "/orders")
+public class OrderController {
+    private static final Logger logger = LoggerFactory.getLogger(OrderController.class);
+    private final OrderRepository orderRepository;
+    public OrderController(OrderRepository orderRepository) {
+        this.orderRepository = orderRepository;
+    }
+
+    @PostMapping
+    public @ResponseBody String createOrder(@RequestBody Order order) {
+        UUID uuid = UUID.randomUUID();
+        order.setId(String.valueOf(uuid));
+        order.setType("order");
```

Comment:
We don't want user to generate a guid for id. Since SDK does not do it, we'll do it here. Also, type will also be order for this flow, so, setting it here.